File: examples/museum/oseg_generated/php/GetSpecialEvent_default_example.php
<?php

require_once __DIR__ . '/vendor/autoload.php';

$config = OpenAPIMuseum\Client\Configuration::getDefaultConfiguration();

try {
    $response = (new OpenAPIMuseum\Client\Api\EventsApi($config))->getSpecialEvent(
        event_id: "dad4bce8-f5cb-4078-a211-995864315e39",
    );

    print_r($response);
} catch (OpenAPIMuseum\Client\ApiException $e) {
    echo 'Exception when calling Events#getSpecialEvent: '
        . print_r($e->getResponseObject());
}

File: examples/museum/oseg_generated/python/GetSpecialEventDefaultExample.py
from datetime import date, datetime
from pprint import pprint

from openapimuseum_client import ApiClient, ApiException, Configuration, api, models

configuration = Configuration()

with ApiClient(configuration) as api_client:
    try:
        response = api.EventsApi(api_client).get_special_event(
            event_id="dad4bce8-f5cb-4078-a211-995864315e39",
        )

        pprint(response)
    except ApiException as e:
        print("Exception when calling Events#get_special_event: %s\n" % e)
